README.md

```diff
@@ -189,6 +189,26 @@ use instance `setConfiguration` method to update configuration of initialized SD
 |defaultDataAttributes|```{ projectId: undefined, languageCodename: undefined }```|Default values for data attributes, which are only used when those data attributes are not found in DOM during data attributes parsing process. For now, only `projectId` and `languageCodename` attributes are supported. |
 |queryParam|`ksl-enabled`|Name of the query parameter that must be present in the URL to turn the smart link injection on. It is not necessary for query parameter to have a truthy value (just the presence of this query parameter is checked). If set to falsy value ('', null), the smart link injection will always be enabled. Query parameter is only used outside Web Spotlight.|
 
+### Customization
+
+The following custom CSS properties can be used to customize the visuals of the SDK output.
```

Comment:
Maybe you could attach this link here:
https://developer.mozilla.org/en-US/docs/Web/CSS/--*
I'm afraid that some users can stuck here because they don't know how to work with CSS custom properties